### ISSUE TITLE
Pilot pause routine should respect if we have run STARTUP or not

### ISSUE THREAD

**Issue by martinjohndyer** [GOTO-OBS/gtecs-control#244]
See what I wrote on Slack:
> When the pilot started it saw we were in bad conditions. So it went into paused mode, which means it parks the mount and closes the dome. Obviously because we've just started both were already true. But it also set the internal flag for the hardware checks that the mount should be parked. Then STARTUP ran, and unparked the mount as the final thing it does. However as it's an external process it can't set the internal pilot hardware check status. So the mount was unparked but the pilot still though it should be parked. Once STARTUP is complete the hardware checks start, and the first thing it sees is that the mount should be parked but isn't. So it reports a hardware error, stops the current process and sends the recovery command - mnt park. That worked, and now everything's happy. But because we started late DARKS began immediately after STARTUP finished, and therefore it was cancelled when the hardware immediately reported a fault. Normally there would have been plenty of time to fix the issue before the first script ran. The solution would be to make the pause command check if we have initialised yet (run STARTUP) and if not don't bother sending the mount park command as we should already be parked.
